compiler/llvm/clangUtil.cpp

```diff
@@ -2385,14 +2385,18 @@ void runClang(const char* just_parse_filename) {
   bool parseOnly = (just_parse_filename != NULL);
   static bool is_installed_fatal_error_handler = false;
 
-  const char* clang_warn[] = {"-Wall", "-Werror", "-Wpointer-arith",
-                              "-Wwrite-strings", "-Wno-strict-aliasing",
-                              "-Wmissing-declarations", "-Wmissing-prototypes",
-                              "-Wstrict-prototypes",
-                              "-Wmissing-format-attribute",
-                              // clang can't tell which functions we use
-                              "-Wno-unused-function",
-                              NULL};
+  // These warnings are _required_ to make sure the code Clang generates
+  // will play well with our backend.
```

Comment:
Could you add a couple of words saying that this is about our `extern` blocks? At the first glance this looked to me like it was talking about Chapel-generated code.